google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionOptions.java

```diff
@@ -634,6 +641,12 @@ static String parseOptimizerVersion(String uri) {
     return value != null ? value : DEFAULT_OPTIMIZER_VERSION;
   }
 
+  @VisibleForTesting
+  static boolean parseReturnCommitStats(String uri) {
+    String value = parseUriProperty(uri, RETURN_COMMIT_STATS_PROPERTY_NAME);
+    return value != null ? Boolean.valueOf(value) : DEFAULT_RETURN_COMMIT_STATS;
```

Comment:
`false` is clearer than a constant whose value is 600 lines up.

Reply:
While it would make it clearer here, it also makes it harder to get an overview of what the all the default values of a new connection is, as you would have to scroll through the different parse methods. Now the default values for all connection properties are grouped in one place.

Reply:
That sounds like a good case for docs or comments.

Reply:
Done